apps/qa/src/pages/pluto/components/source_data_versions_report.py

```diff
@@ -12,6 +12,8 @@
     "pluto_pts": "Department of Finance - Property Tax System (PTS)",
     "lpc_historic_districts": "Landmarks Preservation Commission - Historic Districts",
     "lpc_landmarks": "Landmarks Preservation Commission - Individual Landmarks",
+    "pluto_input_numbldgs": "Office of Technology & Innovation – Building Footprint Centroids",  # this dataset is based on OTI building footprints
+    "dpr_greenthumb": "Department of Parks and Recreation – GreenThumb Garden Info",
```

Comment:
Looking forward to moving to ingest so we can have both "dataset_id" and "dataset_name" in source_data_versions.csv

For this - I can foresee a future where another source dataset gets added or something and we forget to add this. Instead of filtering in line 52, should we just rename the datasets if they're found in source name mappings? Something like

```python
df["dataset"] = df["dataset"].apply(lambda d: SOURCE_NAME_MAPPINGS.get(d, d))
```

So if we forgot to have "dof_dtm" in the name mappings, it'll still show up in the table as "dof_dtm" instead of being missing entirely

Reply:
In the QA page, we have a note about this source table that only a subset of sources are listed, which GIS copies intro pluto readme... So not sure about a case where we would be adding sources implicitly

Reply:
Gotcha. If this serves a more specific purpose, then that makes sense